.claude/commands/polish.md

```
@@ -0,0 +1,185 @@
+---
+description: Review and address issues in existing code - fix problems or justify skipping
+argument-hint: [branch, PR#, file, or area to polish]
+---
+
+## General Guidelines
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)